src/translations/constructor.translations.js

```diff
@@ -1,32 +1,8 @@
 export const CONSTRUCTOR_TITLES = {
   0: {
-    MODEL: 'Модель',
-    BASIC: 'Основа',
-    PATTERN: 'Гобелен',
-    BOTTOM: 'Низ',
-    SIZE: 'Розмір',
-    DEFAULT_PRICE: 'Базова ціна товару:',
-    TOTAL_PRICE: 'Загальна вартість',
-    END_PRICE: 'Кінцева ціна: ',
-    DEFAULT_PRICE_VALUE: '1400 ',
-    MORE_OPTIONS: 'Більше опцій'
+    DEFAULT_PRICE_VALUE: '1400 '
```

Comment:
please move this price value to the index file in configs because it isn't a translation

Reply:
Done